pkg/snapstore/snapshot.go

```diff
@@ -122,9 +124,15 @@ func ParseSnapshot(snapPath string) (*brtypes.Snapshot, error) {
 	}
 
 	//parse creation time as well as parse the Snapshot compression suffix
-	timeWithSnapSuffix := strings.Split(tokens[3], ".")
-	if len(timeWithSnapSuffix) == 2 {
-		s.CompressionSuffix = "." + timeWithSnapSuffix[1]
+	lastNameToken := strings.Split(tokens[3], "/")
```

Comment:
why you are splitting at ` "/"` , Isn't you can directly split at `" . "` ?

Reply:
No. There was a bug here that I fixed by splitting on "/". Otherwise, for snapnames that end with "/something" `timeWithSnapSuffix" would also contain this "/something" and the subsequent logic for determining the suffix would fail.

Reply:
Thanks to clarify but ideally snapname shouldn’t have `"/"` . Do you know why it is having `"/"` in snapname ?

Reply:
It always does for "chunk" snapshots.